training_scripts/gsgnn_np/ml_nc_utext_roberta.yaml

```diff
@@ -0,0 +1,53 @@
+---
+version: 1.0
+lm_model:
+  node_lm_models:
+    -
+      lm_type: roberta
+      model_name: "roberta-base"
```

Comment:
can you specify the model name in the command argument instead of creating a new yaml file?

Reply:
I think we do not support changing lm_model configures through command line.